app/controllers/transaction_controller.rb

```diff
@@ -1,15 +1,50 @@
+
 class TransactionController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    @base = Transbank::Onepay::Base::DEFAULT_CALLBACK
+  end
 
   def create
+    Transbank::Onepay::Base.shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+    Transbank::Onepay::Base.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
+
+    Transbank::Onepay::Base.integration_type = 'TEST'
+    sample_data = {"items":[
+                            {"amount":36000,"quantity": 1,"description":"Fresh Strawberries"},
+                            {"amount":16000,"quantity":1,"description":"Lightweight Jacket"}
+                            ]}
+    cart = Transbank::Onepay::ShoppingCart.new sample_data['items']
```

Comment:
why `sample_data['items']`? Why not just `items`?. Seems like the outer hash is serving no purpose

Reply:
Also: Can't the keys be symbols? They have to be strings? Why?